### ISSUE TITLE
What's 'If sentances'?

### ISSUE THREAD

**Issue by bcm-at-zama** [zama-ai/fhevm-solidity#677]
maybe it's a computer science word I've not heard of nor found on google. I would have said "If statements" or "Dealing with conditions" or "Control flow"

**Comment by poppyseedDev** [comment]
You are correct if statements is more correct or "Dealing with conditions" or "Control flow", you can update docs with any you wish.
